### PR TITLE
Unused parameter in function and  Adding DeepSource 

### PR DESCRIPTION
**Pull request by Cvr421**
## Describe your changes
  Unused parameter should be replaced by underscore into the function 
  Adding Deepsource to make things easier to analyse the code 
## What is DeepSource and How to use the DeepSource ?
      DeepSource is an automated code review tool that helps developers find and automatically fix bugs in their code.
      And most important it's save  the developer time
   How to use ?
       1.  Create an account on DeepSource.
       2.  Pick a Python repository to analyse.
       3.  Visit to DeepSource dashboard.
       4.  Choose your project from the repository list.
       5.  Add DeepSource configuration file.
       6.  Run the analysis.
    Note: It will shows the all kind of different  bugs and suitable fixes 
  
## Screenshots - If Any (Optional)
   
deep
* VISIT DEEPSOURCE DASHBOARD
Add github repository
* SELECT LANGUAGE TO FIX BUGS
Add the language of the project
* LANGUAGE AND FRAMEWORKS
Language
* THEN CODE ANALYSIS REPORT
Report 4

## Issue ticket number and link

   696 
 
## Checklist before requesting a review
- [✓ ] My code follows the style guidelines of this project
- [✓] I have commented my code, particularly in hard-to-understand areas
- [✓] I have performed a self-review of my code
- [✓] If it is a core feature, I have added thorough tests.
- [✓] New and existing unit tests pass locally with my changes

**Please open the PR against the `dev` branch (Unless the PR contains only documentation changes)**

### CODE REVIEW

File: .deepsource.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version = 1
```
Comment (dwertent) [inline .deepsource.toml:1]: Please add to the PR a description of how to use the `deepsource`.
How about adding it to the github actions?

**Review by dwertent — CHANGES_REQUESTED** [review]

File: .deepsource.toml (hunk 1 above)
Comment (Cvr421) [reply .deepsource.toml:1]: Hello sir i am a beginner in open source and i am  learning this first time so please sir guide me so i can do better

Comment (Cvr421) [reply .deepsource.toml:1]: It will make things easier if add to the github action

Comment (Cvr421) [reply .deepsource.toml:1]: Ok i will add How to use deepsource in PR Discription
 Thank you so much for guiding me

Comment (Cvr421) [reply .deepsource.toml:1]: Hello sir as per your guidence i have updated the changes on how to use DeepSource
Please tell me know if i can do anything else 
Thank you

**Comment by Cvr421** [comment]
Hello @dwertent  as you requested to make some changes and that i have done weeks before please review again

**Comment by dwertent** [comment]
@Cvr421  Will the graph update automatically when we make changes? If not, what can we do so it will get updated?

**Comment by Cvr421** [comment]
@dwertent Did you mean that when we make any changes or fix bugs. That will show in Deepsource  analyzer and update the graph .  Is that i am getting you right

**Comment by Cvr421** [comment]
@dwertent 
It will update the graph when we create the pull request directly from deep source account. when bugs are fixed

**Comment by Cvr421** [comment]
@dwertent Please review again

**Comment by dwertent** [comment]
This should be broken down into two PRs, one for unused function, and the other for deepsource integration.
For now, I do not want to add deepsource integration since it is not clear to me how it will get updated without manually changing the badge.

**Pull request closed without merging**